ontology/proco.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: ontology_detail
+id: proco
+title: "PROCO: Ontology of Process Chemistry"
+contact:
+  email: wes_schafer@merck.com
+  label: Wes Schafer
+  github: schaferw
+  orcid: ?
+description: OPC covers process chemistry, the chemical field concerned with scaling up laboratory syntheses to commercially viable processes.
+domain: biochemistry
+homepage: https://github.com/proco-ontology/PROCO
+products:
+  - id: proco.owl
+dependencies:
+  - id: ro
+  - id: obi
+  - id: chebi
+  - id: pato
+  - id: cheminf
+  - id: sbo
+tracker: https://github.com/proco-ontology/PROCO/issues
+license:
+  url: https://creativecommons.org/licenses/by/4.0/
+  label: CC BY 4.0
+usages:
+  - user: Merck & Co.
+    description: pharmaceutical API development
```

Comment:
If you want usages to be listed (not mandatory), you need to provide actual evidence like: https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/ontology/mondo.md?plain=1#L50

So basically a page somewhere that shows how a specific term of PROCO is used by the `user` in this record.

Reply:
Currently the usages are internal, proprietary ontologies/databases.    Publishing in OBO should provide public examples as well as on-going discussions with public chemical databases about using the ontology.   A manuscript is also in-progress

Reply:
Ok, this means I will for now remove the usages. You can always add them later when public evidence becomes available!